package.json

```diff
@@ -21,6 +21,7 @@
     "gulp-ruby-sass": "^1.0.1",
     "gulp-sass": "^1.3.3",
     "gulp-size": "^1.2.1",
-    "gulp-uglify": "^1.1.0"
+    "gulp-uglify": "^1.1.0",
+    "vinyl-source-stream": "^1.1.0"
```

Comment:
I had to add this to get mine to build...not sure if it was just on my machine or what...